Header header                        # header for time/frame information
string ns                            # Namespace to place this object in... used in conjunction with id to create a unique name for the object
int32 id                             # object ID

string filename
roi_and_speed_2d[] rois2d
roi_and_speed_3d[] rois3d
int32[] prevRois